plugins/woocommerce-blocks/assets/js/base/components/reviews/review-list-item/index.tsx

```diff
@@ -186,9 +202,10 @@ const ReviewListItem = ( { attributes, review = {} }: ReviewListItemProps ) => {
 						showReviewRating ||
 						showReviewDate ) && (
 						<div className="wc-block-review-list-item__meta wc-block-components-review-list-item__meta">
-							{ showReviewRating && getReviewRating( review ) }
+							{ showReviewRating &&
+								getReviewRating( review, reviewRatingId ) }
 							{ showProductName &&
-								getReviewProductName( review ) }
+								getReviewProductName( review, reviewRatingId ) }
```

Comment:
Could you explain why the product name is described by the star rating ID? Why does this need to be described by anything, isn't the product name itself fine?

Reply:
@opr I explained it in detail in the [issue's thread].(https://github.com/woocommerce/woocommerce/issues/52133#issuecomment-2610194382).

Please let me know if you have any questions 😉